resources/spec.yaml

```diff
@@ -10598,6 +10598,16 @@ definitions:
         operationId: optOutUsersFromOktaCommunicationEmails
     x-okta-tags:
       - Org
+    x-okta-crud:
+      - alias: read
+        arguments: []
+        operationId: getOktaCommunicationSettings
+      - alias: optin
+        arguments: []
+        operationId: optInUsersToOktaCommunicationEmails
+      - alias: optout
+        arguments: []
+        operationId: optOutUsersFromOktaCommunicationEmails
```

Comment:
`optin` and `optout` are not CRUD-type operations. Also `optInUsersToOktaCommunicationEmails` and `optOutUsersFromOktaCommunicationEmails` already listed in `x-okta-operations` section below.